dbt_airflow_factory/builder.py

```diff
@@ -32,12 +33,31 @@ def _load_dbt_manifest(manifest_path: str) -> dict:
             logging.debug("Manifest content: " + str(manifest_content))
             return manifest_content
 
-    def _make_dbt_test_task(self, model_name: str, is_in_task_group: bool) -> BaseOperator:
+    @staticmethod
+    def _get_dbt_version(manifest: dict) -> version.Version:
+        return version.parse(manifest["metadata"]["dbt_version"])
```

Comment:
Why is it needed?

Reply:
`dbt >= 1.0.0` has different option for indirect tests than `dbt >= 0.20, <1.0.0`